config/src/test/java/org/springframework/security/config/web/server/OAuth2ResourceServerSpecTests.java

```diff
@@ -429,6 +447,38 @@ ReactiveAuthenticationManager authenticationManager() {
 		}
 	}
 
+	@EnableWebFlux
+	@EnableWebFluxSecurity
+	static class CustomBearerTokenServerAuthenticationConverter {
+		@Bean
+		SecurityWebFilterChain springSecurity(ServerHttpSecurity http) throws Exception {
+			// @formatter:off
+			http
+				.authorizeExchange()
+					.anyExchange().access(authorizationManager())
```

Comment:
Is there a reason you need this extra instrumentation, as opposed to something like 

```suggestion
					.anyExchange().access("SCOPE_message:read")
```

If we can, it's nice to be able to reduce the number of moving parts in a test.

Reply:
You are correct I can switch this test. I will issue a new commit that changes this.